workspaces/arborist/test/query-selector-all.js

```diff
@@ -24,6 +24,7 @@ t.test('query-selector-all', async t => {
     │   └── lorem@1.0.0 (production dep of baz)
     ├── abbrev@1.1.1 (production dep of query-selector-all-tests)
     ├─┬ b@1.0.0 -> ./b (workspace)
+    │ ├── a@2.0.0 (dev dep of b, deduped)
```

Comment:
It wasn't previously represented here, but the test fixture does have the `a` workspace as a dependency of `b`